pkg/yurtappmanager/webhook/add_uniteddeployment.go

```diff
@@ -22,9 +22,18 @@ import (
 	"github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/util/gate"
 	"github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/webhook/uniteddeployment/mutating"
 	"github.com/openyurtio/yurt-app-manager/pkg/yurtappmanager/webhook/uniteddeployment/validating"
+	"k8s.io/kubernetes/pkg/capabilities"
 )
 
 func init() {
+	capabilities.Initialize(capabilities.Capabilities{
```

Comment:
I suggest that you put this code logic in init () method of  `pkg/yurtappmanager/webhook/server.go` file.
Because `yurtappDaemons` may have this problem in the future, it is common.

Reply:
Done